pkg/process/config/config_test.go

```diff
@@ -410,60 +402,6 @@ func TestProxyEnv(t *testing.T) {
 	os.Unsetenv("PROXY_PASSWORD")
 }
 
-func TestEnvSiteConfig(t *testing.T) {
-	newConfig()
-	defer restoreGlobalConfig()
-
-	assert := assert.New(t)
-
-	newConfig()
-	agentConfig := loadAgentConfigForTest(t, "./testdata/TestEnvSiteConfig.yaml", "")
-	assert.Equal("process.datadoghq.io", agentConfig.APIEndpoints[0].Endpoint.Hostname())
-
-	newConfig()
-	agentConfig = loadAgentConfigForTest(t, "./testdata/TestEnvSiteConfig-2.yaml", "")
-	assert.Equal("process.datadoghq.eu", agentConfig.APIEndpoints[0].Endpoint.Hostname())
-
-	newConfig()
-	agentConfig = loadAgentConfigForTest(t, "./testdata/TestEnvSiteConfig-3.yaml", "")
-	assert.Equal("burrito.com", agentConfig.APIEndpoints[0].Endpoint.Hostname())
-
-	newConfig()
-	os.Setenv("DD_PROCESS_AGENT_URL", "https://test.com")
-	agentConfig = loadAgentConfigForTest(t, "./testdata/TestEnvSiteConfig-3.yaml", "")
-	assert.Equal("test.com", agentConfig.APIEndpoints[0].Endpoint.Hostname())
-	os.Unsetenv("DD_PROCESS_AGENT_URL")
-
-	newConfig()
-	err := os.Setenv("DD_PROCESS_AGENT_DISCOVERY_ENABLED", "true")
-	require.NoError(t, err)
-	agentConfig = loadAgentConfigForTest(t, "./testdata/TestEnvSiteConfig-ProcessDiscovery.yaml", "")
-	require.NoError(t, err)
-	os.Unsetenv("DD_PROCESS_AGENT_DISCOVERY_ENABLED")
-}
-
-func TestEnvProcessAdditionalEndpoints(t *testing.T) {
-	newConfig()
-	defer restoreGlobalConfig()
-
-	assert := assert.New(t)
-
-	expected := make(map[string]string)
-	expected["key1"] = "url1.com"
-	expected["key2"] = "url2.com"
-	expected["key3"] = "url2.com"
-	expected["apikey_20"] = "my-process-app.datadoghq.com" // from config file
-
-	os.Setenv("DD_PROCESS_ADDITIONAL_ENDPOINTS", `{"https://url1.com": ["key1"], "https://url2.com": ["key2", "key3"]}`)
-	defer os.Unsetenv("DD_PROCESS_ADDITIONAL_ENDPOINTS")
```

Comment:
It looks like that this settings is testing `DD_PROCESS_ADDITIONAL_ENDPOINTS` that we're not deprecating, right?

https://github.com/DataDog/datadog-agent/blob/main/pkg/process/config/config.go#L324-L331

Reply:
Oh thank you for pointing that out, I forgot to include that in `process.go`!